snuba/clusters/storage_sets.py

```diff
@@ -20,6 +20,7 @@
     "EVENTS_ANALYTICS_PLATFORM": "events_analytics_platform",
     "GROUP_ATTRIBUTES": "group_attributes",
     "PROFILE_CHUNKS": "profile_chunks",
+    "UPTIME_RESULTS": "uptime_monitor_checks",
```

Comment:
You don't need this. The migration belongs to the `events_analytics_platform` storage set already. That also means removing migration related changes in this PR (in `group_loader.py`, `groups.py` and the directory in `snuba_migrations`).

I believe it hasn't been run because it's prefixed with `0020_` when the previous one is `0018_`, and yes, that's my fault since I reverted `0019_`, but as soon as you fixed that, the migration will run everywhere the `events_analytics_platform` is available.

Reply:
i'll update the migration in a separate PR! thanks for the callout.

Reply:
i've also removed the migration related changes as specified. thanks.